ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -162,7 +208,15 @@ async def handle_dispatch(self, websocket, path):
                         f"ignoring since I am {self._ee_id}"
                     )
                     continue
-                await self._dispatcher.handle_event(event)
+                try:
+                    await self._dispatcher.handle_event(event)
+                except BaseException:
+                    logger.warning(
+                        "cannot handle event - closing connection to dispatcher"
+                    )
+                    websocket.close()
+                    return
```

Comment:
```suggestion
                except BaseException as ex:
                    logger.exception(
                        "cannot handle event - closing connection to dispatcher"
                    )
                    websocket.close(code=1011, reason=f"cannot handle event: {ex}")
                    return
```
Maybe omit `ex` from `reason`, but give one.

Reply:
Good catch! Will do.